demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -504,6 +506,21 @@ def _is_approved_usecases(self) -> bool:
                 return False
         return True
 
+    def _is_version_format(self, version: str) -> bool:
```

Comment:
Improve function name; the current one doesn't mean anything.
```suggestion
    def _is_version_format_valid(self, version: str) -> bool:
```